html/Task_5/style.css

```diff
@@ -0,0 +1,222 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Page title */
+.page-title {
+  margin: 0;
+  padding: 20px 0;
+  text-align: center;
+}
+
+.page-title--font {
+  font-family: "Kaushan Script", cursive;
+  font-size: 45px;
+}
+
+.page-title--border {
+  border: lightgray 1px solid;
```

Comment:
as you don't have any other page titles you can just use one class, it will be cleaner and more logic, you need modifiers if something looks not like typical element, it applies to all other places where you unnecessary create modifiers